nodejs/webrtc-video-meeting/public/webrtc_mgr.js

```diff
@@ -172,27 +172,35 @@ async function screenShare() {
     my_screen_stream = null;
     document.getElementById("share").style.display = "none";
   } else {
-    // we're not sharing, so start, and update the text of the link
-    document.getElementById("screen_share").innerHTML = "Stop Sharing";
-
     video_constraints = {
       frameRate: 30,
     };
     // getDisplayMedia is the magic function for screen/window/tab sharing
-    my_screen_stream = await navigator.mediaDevices.getDisplayMedia({
-      audio: false,
-      video: video_constraints,
-    });
+    try {
+      my_screen_stream = await navigator.mediaDevices.getDisplayMedia({
+        audio: false,
+        video: video_constraints,
+      });
+    } catch(err) {
+      if (err.name != "NotAllowedError") {
```

Comment:
If you go into the screen sharing dialog and cancel out without picking anything, you get a "NotAllowedError"